frontend/hub/collections/CollectionDetails.tsx

```diff
@@ -368,16 +369,24 @@ function CollectionDocumentationTab(props: { collection?: CollectionVersionSearc
 
   const [content, setContent] = useState<IContents>();
 
-  const { data } = useGet<CollectionDocs>(
-    hubAPI`/_ui/v1/repo/published/${collection?.collection_version?.name || ''}/${
-      collection?.collection_version?.name || ''
-    }/?include_related=my_permissions`
+  const { data } = useGet<CollectionVersionListResponse>(
+    pulpAPI`/content/ansible/collection_versions/?namespace=${
+      collection?.collection_version?.namespace || ''
+    }&name=${collection?.collection_version?.name || ''}&version=${
+      collection?.collection_version?.version || ''
+    }&offset=0&limit=1`
   );
 
+  const readme = useMemo(() => {
```

Comment:
@himdel Do you think this useMemo is necessary? The data comes from useGet, which memoizes the data anyway, so direct reference to the readme blob should be always the same.

Reply:
oh, good catch, but moot point, `readme` itself is unused :) we can remove the whole block